### ISSUE TITLE
bestrefs "--sync-references=1" option broken

### ISSUE THREAD

**Issue by sean-lockwood** [spacetelescope/crds#177]
In crds v7.0.10 of bestrefs, the --sync-references=1 option raises an error.  This error is not produced in v7.0.7, or if the --sync-references=1 option is left out.

```
(recent) bash-3.2$ python -m crds.bestrefs --update-bestrefs --files obpz06030_wav.fits 
CRDS - INFO -  No comparison context or source comparison requested.
CRDS - INFO -  ===> Processing obpz06030_wav.fits
CRDS - INFO -  0 errors
CRDS - INFO -  0 warnings
CRDS - INFO -  2 infos

(recent) bash-3.2$ python -m crds.bestrefs --update-bestrefs --sync-references=1 --files obpz06030_wav.fits 
CRDS - INFO -  No comparison context or source comparison requested.
CRDS - INFO -  ===> Processing obpz06030_wav.fits
Traceback (most recent call last):
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/bestrefs.py", line 1332, in <module>
    sys.exit(main())
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/bestrefs.py", line 1327, in main
    errors = BestrefsScript()()
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/cmdline.py", line 174, in __call__
    self._main()
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/cmdline.py", line 155, in _main
    self._exit_status = self.main()
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/bestrefs.py", line 1000, in main
    self.post_processing()
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/bestrefs.py", line 1223, in post_processing
    self.sync_references()
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/bestrefs.py", line 1302, in sync_references
    synced_references = {tup.new for tup in self.updates if tup.new not in ["N/A"]}
  File "/Users/lockwood/anaconda/envs/recent/lib/python2.7/site-packages/crds/bestrefs.py", line 1302, in <setcomp>
    synced_references = {tup.new for tup in self.updates if tup.new not in ["N/A"]}
AttributeError: 'str' object has no attribute 'new'

**Comment by jaytmiller** [comment]
Hi Sean,

I was able to reproduce the problem and have a fix.   Hopefully it will be on spacetelescope master in a couple days once JWST Build-7 is truly frozen.    

Meanwhile,  if you need the fix now there's a branch in my personal github here:

https://github.com/jaytmiller/crds/tree/bestrefs-sync-references-fix

Thanks,
Todd

**Comment by jaytmiller** [comment]
This issue is now addressed with CRDS release 7.1.0 available via astroconda-contrib, github, and pip.
